core/services/ocr2/plugins/ccip/ccipexec/initializers.go

```diff
@@ -151,7 +160,8 @@ func jobSpecToExecPluginConfig(lggr logger.Logger, jb job.Job, chainSet legacyev
 }
 
 func NewExecutionServices(lggr logger.Logger, jb job.Job, chainSet legacyevm.LegacyChainContainer, new bool, argsNoPlugin libocr2.OCR2OracleArgs, logError func(string), qopts ...pg.QOpt) ([]job.ServiceCtx, error) {
-	execPluginConfig, backfillArgs, err := jobSpecToExecPluginConfig(lggr, jb, chainSet)
+	ctx := context.Background()
+	execPluginConfig, backfillArgs, err := jobSpecToExecPluginConfig(ctx, lggr, jb, chainSet)
```

Comment:
Since the context is only being passed into this function this is more style-ish:
```suggestion
	execPluginConfig, backfillArgs, err := jobSpecToExecPluginConfig(context.Background(), lggr, jb, chainSet)
```